docs/CSBR.md

```diff
@@ -792,7 +793,7 @@ The CA SHALL reject a certificate request if the requested Public Key does not m
 
 ### 6.1.2 Private key delivery to subscriber
 
-If the CA or any Delegated Third Party is generating the Private Key on behalf of the Subscriber where the Private Keys will be transported to the Subscriber outside of the Signing Service's secure infrastructure, then the entity generating the Private Key MUST either transport the Private Key in hardware with an activation method that is equivalent to 128 bits of encryption or encrypt the Private Key with at least 128 bits of encryption strength. Allowed methods include using a 128-bit AES key to wrap the private key or storing the key in a PKCS 12 file encrypted with a randomly generated password of more than 16 characters containing uppercase letters, lowercase letters, numbers, and symbols for transport.
+If the CA or any Delegated Third Party is generating the Private Key on behalf of the Subscriber where the Private Keys will be transported to the Subscriber outside of the CA's or Delegated Third Party's secure infrastructure, then the entity generating the Private Key MUST either transport the Private Key in hardware with an activation method that is equivalent to 128 bits of encryption or encrypt the Private Key with at least 128 bits of encryption strength. Allowed methods include using a 128-bit AES key to wrap the Private Key or storing the Private Key in a PKCS 12 file encrypted with a randomly generated password of more than 16 characters containing uppercase letters, lowercase letters, numbers, and symbols for transport.

```

Comment:
Why do we explicitly call out a "Delegated Third Party" in this section? Which scenario are we trying to allow?

Reply:
I think we are just adding "Delegated Third Party" to be complete to describe the "secure infrastructure."

Reply:
We are replacing "Signing Service" with "Delegated Third Party" which is a bit strange and I would like to discuss a bit more to understand the possible scenarios we want to support.

Reply:
Follwing-up on the meeting of 2022-12-01, add a new sentence:

"If a Signing Service is generating a Private Key on behalf of the Subscriber, that Private Key SHALL NOT be transported to the Subscriber".

Reply:
Sending subscriber keys as PKCS#12 files seems very sketchy. Doesn't that bypass the whole subscriber private key protection requirements? How does the CA verify that the subscriber is using an HSM when the CA simply sent a P12 file? Requre and audit that the key was imported in an HSM and then the P12 file was wiped? 
Today signing keys are laying around on P12 files, isn't that what we want to get away from? 
PKCS#12 is not a well defined protection btw, different encryption algorithms can be used, including single DES, RC2, etc.

Reply:
I think the purpose of this section was to transport keys generated in software. Perhaps we should only allow this section until 1 June 2023, when the private key protection ballot is in place. We could add the text which Dimitris provided. We could also state something to the effect that a CA can transport private keys to the Subscriber in a Hardware Crypto Module, where the activation data is provided out of band. This would line up with 6.2.7.4.2 item 1.